drivers/windows/R/mounts.R

```diff
@@ -1,3 +1,5 @@
+##' @importFrom utils read.csv
```

Comment:
I don't think we want this generally, but if we do it should be above the function that uses `read.csv` not the top of the file. But just tweak the function in question to quieten the note

Reply:
Done